### PR TITLE
my first commit using the CLI

### PR DESCRIPTION
**Pull request by wisiwii**
# Updated caption

- [ ] Code
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: _slides/06-wisiwii.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+#**Wisiwii**
```
Comment (amartinlaso) [inline _slides/06-wisiwii.md:6]: Good 🔥

**Review by amartinlaso — CHANGES_REQUESTED** [review]
Perfect! 👍

File: _slides/06-wisiwii.md (hunk 1 above)
Comment (wisiwii) [reply _slides/06-wisiwii.md:6]: thx!

Comment (amartinlaso) [inline _slides/06-wisiwii.md:6]:
```suggestion
#**Hi Haaland**
```

**Review by amartinlaso — APPROVED** [review]

**Pull request merged**